resources/function_help/json/Sensors

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Sensors",
+  "type": "group",
+  "description": "This group contains functions to operate on sensors."
```

Comment:
@nirvn If ever you have a more helpful wording...

Reply:
@DelazJ , thanks for this. I'd say maybe this:

```suggestion
  "description": "This group contains functions to interact with sensors."
```